@codex please fix

```
22s
Run python desktop-tauri/scripts/test_packaged_operator_e2e.py
Traceback (most recent call last):
  File "D:\a\token.place\token.place\desktop-tauri\scripts\test_packaged_operator_e2e.py", line 210, in <module>
    raise SystemExit(main())
                     ^^^^^^
  File "D:\a\token.place\token.place\desktop-tauri\scripts\test_packaged_operator_e2e.py", line 186, in main
    raise RuntimeError(
RuntimeError: bridge did not emit started/running event; output=
Error: Process completed with exit code 1.
```

new codex task, not a r/e/v/i/e/w task

--- a/desktop-tauri/scripts/test_packaged_operator_e2e.py
+++ b/desktop-tauri/scripts/test_packaged_operator_e2e.py
@@ -73,10 +73,12 @@ def create_packaged_layout(tmp_root: Path) -> Path:
 
 
 def enqueue_bridge_stdout(stdout: object, output_queue: queue.Queue[bytes]) -> None:
-    if not hasattr(stdout, "read"):
+    if not hasattr(stdout, "readline"):
         return
+
+    readline = stdout.readline
     while True:
-        chunk = stdout.read(4096)
+        chunk = readline()
         if not chunk:
             return
         output_queue.put(chunk)

